packages/app/src/utils/sentry.ts

```
@@ -13,6 +13,12 @@ Sentry.init({
   replaysSessionSampleRate: 0,
   replaysOnErrorSampleRate: 0,
   tracePropagationTargets: [],
+  ignoreErrors: [
+    'User rejected methods',
+    'User disapproved requested methods',
+    'WebSocket request failed',
```

we def don't want to ignore errors releated to websockets for now as they are crashing the app.